www/front_src/src/Resources/Filter/index.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { useState } from 'react';
+
+import { Paper, Grid, Typography, Button, makeStyles } from '@material-ui/core';
+
+import {
+  AutocompleteField,
+  ConnectedAutocompleteField,
+  SelectField,
+  SearchField,
+  SelectEntry,
+} from '@centreon/ui';
+
+import {
+  labelFilter,
+  labelStateFilter,
+  labelResourceName,
+  labelSearch,
+  labelTypeOfResource,
+  labelState,
+  labelStatus,
+  labelHostGroup,
+  labelServiceGroup,
+  labelClearAll,
+} from '../translatedLabels';
+import {
+  unhandledProblemsFilter,
+  resourceProblemsFilter,
+  allFilter,
+  states,
+  resourceTypes,
+  statuses,
+  Filter as FilterModel,
+  FilterGroup,
+} from './models';
+import SearchHelpTooltip from '../SearchHelpTooltip';
+import {
+  buildHostGroupsEndpoint,
+  buildServiceGroupsEndpoint,
+} from '../api/endpoint';
+
+const useStyles = makeStyles((theme) => ({
+  filterBox: {
+    padding: theme.spacing(2),
+    backgroundColor: theme.palette.common.white,
+  },
+  filterGroup: {
+    minWidth: 250,
+  },
+  autocompleteField: {
+    minWidth: 250,
+    maxWidth: 400,
+  },
+}));
+
+interface Props {
+  filter: FilterGroup;
+  onFilterGroupChange: (event) => void;
+  currentSearch?: string;
+  onSearchRequest: (event) => void;
+  selectedResourceTypes: Array<FilterModel>;
+  onResourceTypesChange: (event, types) => void;
+  selectedStates: Array<FilterModel>;
+  onStatesChange: (_, states) => void;
+  selectedStatuses: Array<FilterModel>;
+  onStatusesChange: (_, statuses) => void;
+  selectedHostGroups?: Array<FilterModel>;
+  onHostGroupsChange: (_, hostGroups) => void;
+  selectedServiceGroups?: Array<FilterModel>;
+  onServiceGroupsChange: (_, serviceGroups) => void;
+  onClearAll: () => void;
+}
+
+const Filter = ({
+  filter,
+  onFilterGroupChange,
+  currentSearch,
+  onSearchRequest,
+  selectedResourceTypes,
+  onResourceTypesChange,
+  selectedStates,
+  onStatesChange,
+  selectedStatuses,
+  onStatusesChange,
+  selectedHostGroups,
+  onHostGroupsChange,
+  selectedServiceGroups,
+  onServiceGroupsChange,
+  onClearAll,
+}: Props): JSX.Element => {
+  const classes = useStyles();
+
+  const [searchFieldValue, setSearchFieldValue] = useState<string>();
+
+  const changeSearchFieldValue = (event): void => {
+    setSearchFieldValue(event.target.value);
+  };
+
+  const requestSearch = (): void => {
+    onSearchRequest(searchFieldValue);
+  };
+
+  const getHostGroupSearchEndpoint = (searchValue): string => {
+    return buildHostGroupsEndpoint({
+      limit: 10,
+      search: `name:^${searchValue}`,
```

Comment:
I'm not sure we want to search Only hostgroups which begin by search value. 
We should remove `^`